lib/test/t_KrigingAlgorithm_std.cxx

```diff
@@ -62,14 +62,12 @@ int main(int, char *[])
       Sample Y2(model(X2));
 
       Basis basis(ConstantBasisFactory(dimension).build());
-      SquaredExponential covarianceModel(Point(1, 1e-05), Point(1, 4.11749));
+      SquaredExponential covarianceModel(Point(1, 1e-02), Point(1, 4.50736));
```

Comment:
Why is this change necessary?

Reply:
Because we dropped the normalization. In practice I will say it is useless as we perform an optimization algorithm. However final results might slightly differ considering the algorithm, the architecture and so. Thus we start from the optimal point.
The point here was obtained using MultiStart and strong convergence criteria (outside this specific test)

Reply:
Ah, of course.